<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QCalendarWidget" name="dsip_kalender">
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(96, 0, 144);
border-left-color: rgb(170, 85, 255);
background-color: rgb(170, 85, 255);
font: 10pt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="2" column="0">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">
font: 24pt &quot;Comic Sans MS&quot;;
color: white;
background-color: rgb(128, 60, 224);</string>
     </property>
     <property name="text">
      <string>Daily Task Planner</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btn_savechanges">
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
color: white;
font:11pt;
background-color: rgb(128, 60, 224);
</string>
     </property>
     <property name="text">
      <string>Save Changes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
